### PR TITLE
fix: Add placement option to date/datetime picker

### PR DESCRIPTION
**Pull request by JKMarkowski**
#### Please provide a link to the associated issue.
fixes https://github.com/SAP/fundamental-ngx/issues/775
#### Please provide a brief summary of this pull request.
I added placement input to date and datetime picker.
#### If this is a new feature, have you updated the documentation?
I added examples on date and datetime docs.

### CODE REVIEW

File: docs/app/documentation/component-docs/date-picker/date-picker-docs.component.html [hunk 1]
```diff
@@ -19,7 +19,8 @@ <h2>Range Date Picker</h2>
 
 <separator></separator>
 <h2>Internationalization</h2>
-<description>It is possible to internationalize both the aria labels and the month names/weekdays through providing a service.</description>
+<description>It is possible to internationalize both the aria labels and the month names/weekdays through providing a service.
+    There is also added <code>[position]</code> attribute which allows us to decide where popup should be shown.</description>
```
Comment (mikerodonnell89) [inline docs/app/documentation/component-docs/date-picker/date-picker-docs.component.html]: this note probably does not belong in the "internationalisation" section

File: docs/app/documentation/component-docs/date-picker/examples/date-picker-i18n-example.component.ts [hunk 2]
```diff
@@ -52,7 +52,7 @@ export class CustomI18nLabels extends CalendarI18nLabels {
 
 @Component({
     selector: 'fd-datepicker-i18n-example',
-    template: `<fd-date-picker [(ngModel)]="selectedDay"></fd-date-picker>`,
+    template: `<fd-date-picker [(ngModel)]="selectedDay" [placement]="'top-end'"></fd-date-picker>`,
```
Comment (mikerodonnell89) [inline docs/app/documentation/component-docs/date-picker/examples/date-picker-i18n-example.component.ts]: Again this probably does not belong in the i18n section but may warrant its own example

**Comment by mikerodonnell89** [comment]
@JKMarkowski would you mind fixing the merge conflicts

**Comment by JKMarkowski** [comment]
@mikerodonnell89 Sure, it's done.

**Pull request merged**